template/code/test-report.code-snip.hbs

```diff
@@ -36,6 +36,43 @@
             <h2 class="card__panel__heading"><span class="heading-char">✓</span> Fix Analysis</h2>
             <div class="card__panel__markdown">{{{markdown ruleiddesc.help.markdown}}}</div>
         </div>
+        {{#if suppression}}
```

Comment:
**praise:** This gating is good and will ensure our other customers do not get the information.